src/frontend/src/components/Mining.vue

```diff
@@ -36,7 +36,7 @@ export default class Mining extends Vue {
   }
 
   onChange() {
-      this.$store.commit('MUTATATION_SET_MINING', this.$data.miningValue);
+      this.$store.commit('MUTATION_SET_MINING', this.$data.miningValue);
```

Comment:
Still can't believe I did this

Reply:
![gif](https://media.tenor.com/images/46f0422b751a738dbb159717a9831671/tenor.gif)